swagger.yaml

```diff
@@ -0,0 +1,515 @@
+    properties:
+      totalCount:
+        type: integer
+        format: int64
+        description: Total count of the objects
+
+  ErrorModel:
+    type: object
+    properties:
+      id:
+        type: string
+        description: unique id identifying the request
+      version:
+        type: string
+      result:
+        type: object
+        properties:
+          success:
+            type: boolean
+          status:
+            description: http status code
+            type: integer
+            format: int32
+          debug:
+            type: object
+          content:
+            type: object
+
+  NewBillingAccount:
+    type: object
+    required:
+       - customerNumber
+       - name
+    properties:
+      customerNumber:
+        type: string
+        description: customer number of billing account
+      name:
+        type: string
+        description: billing account name
+      status:
+        type: string
+        description: BillingAccount status
+      startDate:
+        type: string
+        description: billing account start date, Datetime (GMT)
+      endDate:
+        type: string
+        description: billing account end date, Datetime (GMT)
+
+  NewBillingAccountBodyParam:
+    type: object
+    properties:
+      param:
+        $ref: "#/definitions/NewBillingAccount"
+
+  BillingAccountBodyParam:
+    type: object
+    properties:
+      param:
+        $ref: "#/definitions/BillingAccount"
+
+  BillingAccount:
+    type: object
+    properties:
+      id:
+        description: unique identifier
+        type: integer
+        format: int64
+      customerNumber:
+        type: string
+        description: customer number of billing account
+      name:
+        type: string
+        description: billing account name
+      status:
+        type: string
+        description: billing account status
+      startDate:
+        type: string
+        description: billing account start date, Datetime (GMT)
+      endDate:
+        type: string
+        description: billing account end date, Datetime (GMT)
+      createdAt:
+        type: string
+        description: Datetime (GMT) when task was created
+        readOnly: true
+      createdBy:
+        type: integer
+        format: int64
+        description: READ-ONLY. User who created this task
+        readOnly: true
+      updatedAt:
+        type: string
+        description: READ-ONLY. Datetime (GMT) when task was updated
+        readOnly: true
+      updatedBy:
+        type: integer
+        format: int64
+        description: READ-ONLY. User that last updated this task
+        readOnly: true
+
+  BillingAccountUser:
+    type: object
+    properties:
+      id:
+        description: unique identifier
+        type: integer
+        format: int64
+      name:
+        type: string
+        description: billing account name
+      description:
+        type: string
+        description: BillingAccount description
+
+  BillingAccountResponse:
```

Comment:
@skyhit Can you add challenge budget?  We don't currently have this in the DB, so just mock it for now...